Solution added for August_23_Stream_of_Characters

vikhyatsingh123 opened the pull request:
hey, I just added solution of Stream of characters in cpp.
If any compiler error or wrong answer happens then let me know.

priyanshisharma reviewed (changes requested):
This problem has been claimed by @Valkyrie006 in #1. Please check. if he doesn't submit it in a few days, i will definitely consider this.

Please solve it for the following function header, just like you would submit it on leetcode.
```
class StreamChecker {
public:
    StreamChecker(vector<string>& words) {
        
    }
    
    bool query(char letter) {
        
    }
};

/**
 * Your StreamChecker object will be instantiated and called as such:
 * StreamChecker* obj = new StreamChecker(words);
 * bool param_1 = obj->query(letter);
 */
```

vikhyatsingh123 commented:
ok, but code that I added was accepted by Leetcode. First I compiled and successfully submitted on Leetcode and then I added this code in the repo. So, you can merge it too.

priyanshisharma commented:
Sure @vikhyatsingh123 , but since @Valkyrie006 claimed it first and has also solved it in #50 , I'm closing this. Thanks a lot for your time and effort ✨
